app/api/discount_codes.py

```diff
@@ -23,14 +23,27 @@ class Meta:
         self_view_kwargs = {'id': '<id>'}
         inflect = dasherize
 
-    @validates_schema
-    def validate_quantity(self, data):
-        if 'max_quantity' in data and 'min_quantity' in data:
-            if data['max_quantity'] < data['min_quantity']:
+    @validates_schema(pass_original=True)
+    def validate_quantity(self, data, original_data):
+        original_data_attributes = original_data['data']['attributes']
+        discount_code = DiscountCode.query.filter_by(id=original_data['data']['id']).one()
+        if 'min_quantity' in original_data_attributes:
```

Comment:
Agreed  @poush. 
@LuD1161 I think what poush is trying to ask is why not use `data` itself, what is the need for using for using `original_data`. Either the above comment does not answer or either rephrase it.
Can't a simple check like do the work if 
```
if 'start_date' in data:
```
if only start_date is provided and not end_date